docs/pfsense_installerconfig.md

```diff
@@ -0,0 +1,14 @@
+# Execute post-install script on PFSense
+After successfully installation **installerconfig** script is called if exists. It has defined structure:
```

Comment:
@adamwawrzynski3mdeb please have a look on other markdown documents how they are formatted. There should be new lines separating title headers etc.